### PR TITLE
returning tracking pixels

### PR DESCRIPTION
**Pull request by nwlosinski**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
1. Drop pixel on auction timeout
2. Drop pixel when auction ends

### CODE REVIEW

File: modules/justpremiumBidAdapter.js [hunk 1]
```diff
@@ -54,6 +62,13 @@ export const spec = {
 
     const payloadString = JSON.stringify(payload)
 
+    LAST_PAYLOAD = payload
+    $$PREBID_GLOBAL$$.onEvent('bidWon', (data) => {
```
Comment (jaiminpanchal27) [inline modules/justpremiumBidAdapter.js]: @nwlosinski `$$PREBID_GLOBAL$$` usage is not allowed in adapter except for prebid verion.  You can drop pixels using http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Configure-User-Syncing

Comment (nwlosinski) [reply modules/justpremiumBidAdapter.js]: @jaiminpanchal27 thanks for the link but we didn't add this to sync users. We need to gather some additional information to be able better detects why we lose. If we are not allowed to use `$$PREBID_GLOBAL$$` how we can achieve that? Is there any other way to send some tracking when action ends?

Comment (mkalafior) [reply modules/justpremiumBidAdapter.js]: @jaiminpanchal27 are you able to provide any input here?

Comment (jaiminpanchal27) [reply modules/justpremiumBidAdapter.js]: @nwlosinski Sorry this didn't came to my mind earlier but you can add `onTimeout` and `onBidwon` function to your bidder spec and achieve this. Docs are here http://prebid.org/dev-docs/bidder-adaptor.html#registering-on-timeout

Some of the bidders are using these functions you can check those for reference.

Comment (mkalafior) [reply modules/justpremiumBidAdapter.js]: @jaiminpanchal27 yeah but `onBidWon` listener is executed only when we won the bidding where we wanted to send tracking pixel if other parties did that.
So I understand that there is no way that you can accept this PR in current shape?

Comment (jaiminpanchal27) [reply modules/justpremiumBidAdapter.js]: @mkalafior yes we cannot as of now.

Comment (nwlosinski) [reply modules/justpremiumBidAdapter.js]: @jaiminpanchal27 thanks for your time. For now we removed this part.

**Pull request merged**